packages/server/src/types/validator/methods.ts

```diff
@@ -0,0 +1,28 @@
+/*-
```

Comment:
Perhaps instead of having a single folder (`validator`) with multiple type files, we could consolidate them into one file (`validator.ts`) that hosts all the 'validator-related' types.

Just a thought, but as I mentioned, since these types are all related to the validator logic, it might be cleaner to keep things in one place.

Reply:
We already have an `index.ts` file inside the validator folder, which servers this purpose